db/seeds/development/comments_seed.rb

```diff
@@ -1,13 +1,20 @@
-def generate_comment(student)
+def generate_comment(id, type)
```

Comment:
Let's call these params `commentable_id` and `commentable_type` just to be explicit.